src/normal_form_game.jl

```diff
@@ -188,7 +214,7 @@ valse otherwise.
 """
 function is_best_response(player::Player,
                           own_action::PureAction,
-                          opponents_actions::Union{Action,ActionProfile,Void};
+                          opponents_actions::Union{Action,PureActionProfile,MixedActionProfile,Void};
```

Comment:
It seems that we are using this particular `Union` quite often. Does it make sense to name it using `typealias`?

Also, it is true that `Union{A,Union{B,C}} == Union{A,B,C}` so even if we don't name this particular Union, we should be able to make use of `ActionProfile` to replace `PureActionProfile,MixedActionProfile`